ChangeLog

```diff
@@ -59,13 +23,18 @@
   + tools: Allow more fine grained filtering of fence-history in crm_mon.
   + tools: Allow crm_rule to check some date_specs.
 
+- Fixes for regressions introduced in Pacemaker-2.0.0
+  + controller: properly detect remote node info requests
```

Comment:
Looking at this again I'm thinking something like "tools: ensure crm_node -n/--name always works from Pacemaker Remote node command line" since that's what users would notice